src/fmu/dataio/datastructure/meta/meta.py

```diff
@@ -250,6 +250,19 @@ class Masterdata(BaseModel):
     smda: Smda
 
 
+class FMUdataioSystemInformation(BaseModel):
+    version: str
+
+
+class SystemInformation(BaseModel):
+    fmu_dataio: Optional[FMUdataioSystemInformation] = Field(
+        alias="fmu-dataio", default=None, examples=["1.2.3"]
+    )
+    komodo: Optional[FMUdataioSystemInformation] = Field(
```

Comment:
Why not `FMUkomodoSystemInformation` or something along those lines? (If reusing `FMUdataioSystemInformation` should it have more generic name?)